forum/qa-include/app/format.php

```diff
@@ -657,10 +657,13 @@ function qa_message_html_fields($message, $options=array())
 		$fields['tags'] = 'id="m'.qa_html($message['messageid']).'"';
 
         // Message content
+        $userPoints = qa_db_select_with_pending(qa_db_user_points_selectspec($message['touserid'], true));
+        $showUrlLinks = ($options['showurllinks'] ?? false) && (qa_is_logged_in() || $userPoints['points'] >= 500);

```

Comment:
Można by tę liczbę punktów 500 przenieść do zmiennej, która by swoją nazwą mówiła czym to jest.